index.html

```diff
@@ -16,47 +16,50 @@
             </h3>
         </div>
     </div>
     <br>
     <div class="row">
         <div class="col-sm-offset-2 col-sm-8 col-lg-4">
 
-            <p class="info">These pre-built nightly snapshots allow developers
-            to try Servo and <a href="https://github.com/servo/servo/issues/new">report
-            issues</a> without <a href="https://github.com/servo/servo/blob/master/docs/HACKING_QUICKSTART.md">
-            building Servo locally</a>.
+            <p class="info">
+                These pre-built nightly snapshots allow developers
+                to try Servo and <a href="https://github.com/servo/servo/issues/new">report
+                issues</a> without <a href="https://github.com/servo/servo/blob/master/docs/HACKING_QUICKSTART.md">
+                building Servo locally</a>.
             </p>
 
-            <p class="info"><b>Please don’t log into your bank with Servo just
-yet!</b> Now that we’ve released our first developer preview, we’ll be
-investing in formal security audits and improving our security practices using
-both existing libraries and Rust &ndash; more information coming soon!
+            <p class="info">
+                <b>Please don’t log into your bank with Servo just
+                yet!</b> Now that we’ve released our first developer preview, we’ll be
+                investing in formal security audits and improving our security practices using
+                both existing libraries and Rust &ndash; more information coming soon!
             </p>
         </div>
         <div class="col-sm-8 col-lg-4 col-sm-push-2 col-lg-push-0">
             <span class="col-sm-8 col-sm-push-0">
                 <a class="btn btn-primary btn-large" id="button-windows-exe" role="button"
-                   href="nightly/windows-msvc/servo-latest.exe">Windows Build Installer (64-bit)</a><br>
+                   href="nightly/windows-msvc/servo-latest.exe">Windows EXE (x64)</a><br>
 
                 <a class="btn btn-primary btn-large" id="button-windows-zip" role="button"
-                   href="nightly/windows-msvc/servo-latest.zip">Windows Build (64-bit)</a><br>
+                   href="nightly/windows-msvc/servo-latest.zip">Windows ZIP (x64)</a><br>
 
                 <a class="btn btn-primary btn-large" id="button-uwp" role="button"
-                   href="nightly/uwp/servo-latest.zip">UWP Build</a><br>
+                   href="nightly/uwp/servo-latest.zip">UWP (arm64/x64)</a><br>
 
                 <a class="btn btn-primary btn-large" id="button-macos" role="button"
-                   href="nightly/mac/servo-latest.dmg">macOS Build</a><br>
+                   href="nightly/mac/servo-latest.dmg">macOS</a><br>
 
                 <a class="btn btn-primary btn-large" id="button-linux" role="button"
-                   href="nightly/linux/servo-latest.tar.gz">Linux Build (64-bit)</a><br>
+                   href="nightly/linux/servo-latest.tar.gz">Linux (x64)</a><br>
```

Comment:
What do you think of the text for the download buttons? I was tempted to put "macOS (x64)" for consistency, but I think 32-bit Macs barely exist these days.

Reply:
This seems fine.